cookbook/configuration/apache_router.rst

```diff
@@ -4,8 +4,9 @@
 How to Use the Apache Router
 ============================
 
-Symfony, while fast out of the box, also provides various ways to increase that speed with a little bit of tweaking.
-One of these ways is by letting Apache handle routes directly, rather than using Symfony for this task.
+Symfony, while fast out of the box, also provides various ways to increase that
+speed with a little bit of tweaking. One of these ways is by letting Apache
+handle routes directly, rather than using Symfony for this task.
```

Comment:
I've added a note about not using Apache Router in aedaccd. Please tell me if it's OK. Thanks.